db/migrate/20220913071239_rename_seances_rows.rb

```diff
@@ -0,0 +1,8 @@
+class RenameSeancesRows < ActiveRecord::Migration[7.0]
```

Comment:
Within one PR you could rename those columns in the CreateSeances migration. Just rollback, rename columns and migrate again.